docs/source/kernel-kubernetes.md

```diff
@@ -104,6 +104,10 @@ spec:
         - name: EG_DEFAULT_KERNEL_NAME
           value: "python_kubernetes"
 
+        # Authorization token required for all requests
```

Comment:
Tokens are not required.  As a result, I think these entries should be commented out in the `enterprise-gateway.yaml` file (on which this section is based) similar to how [`volumeMounts`](https://github.com/jupyter/enterprise_gateway/blob/443708ffb4cac466fb43e6daccddcdf24aed1945/etc/kubernetes/enterprise-gateway.yaml#L162-L170) are handled.  Will likely need a placeholder value like `<configured-auth-token>` as the value.